core/src/main/java/org/testcontainers/dockerclient/DockerMachineClientProviderStrategy.java

```diff
@@ -16,6 +16,9 @@
  */
 @Slf4j
 public class DockerMachineClientProviderStrategy extends DockerClientProviderStrategy {
+
+    public static final int PRIORITY = EnvironmentAndSystemPropertyClientProviderStrategy.PRIORITY - 10;
```

Comment:
I actually had to remove `~/.testcontainers.properties` a couple of times because it saved Docker Machine strategy when Docker for Mac was turned off :)

Maybe we can mark Docker Machine strategy as non-persistable, so that it will be used only as a fallback?
IMO it makes sense to do that in 2017 :D